packages/dts-generator/src/js-utils/write-url-to-file.js

```diff
@@ -0,0 +1,28 @@
+import fsextra from "fs-extra";
+const { writeFile } = fsextra;
```

Comment:
like above - can be combined into a single import line

Reply:
done

Reply:
not possible because fs-extra is a commonjs module, see below